api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -17,7 +17,6 @@
 import org.pmiops.workbench.actionaudit.auditors.WorkspaceAuditor;
 import org.pmiops.workbench.billing.FreeTierBillingService;
 import org.pmiops.workbench.cdr.CdrVersionContext;
-import org.pmiops.workbench.cdrselector.WorkspaceResourcesService;
```

Comment:
many of these because I moved it from `cdrselector` ?! to `workspaces.resources`